### PR TITLE
Created courses endpoints

### PR DESCRIPTION
**Pull request by WillisLi**
What we did: Implemented course routes and model to table and updated course migrations and seeds.
Files we added:  courseModel.js, courseRouter.js
Why were the changes made: To push up some endpoints for frontend.

Co-authored-by: Andrew Cummings <andrew.amalee@gmail.com>
[Loom Video](https://www.loom.com/share/8caa4f8ac67e42cf9ddf110e22d11f4a)

### CODE REVIEW

File: data/migrations/20211102124951_inboxes.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ exports.up = (knex) => {
       table.timestamp('sent_at').defaultTo(knex.fn.now());
       table.string('title').notNullable();
       table.boolean('read').defaultTo(false);
-      table.text('message').notNullable();2
+      table.text('message').notNullable();
+      2;
```
Comment (CptHappyHands) [inline data/migrations/20211102124951_inboxes.js]: Remove the 2. Pretty sure it was just a typo

File: api/course/courseModel.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const db = require('../../data/db-config');
+
```
Comment (MikeLikesCode) [inline api/course/courseModel.js:2]: Nice names for the DB functions. This will definitely help the other teams in the future.

**Review by MikeLikesCode — APPROVED** [review]

**Review by JoeyMBrown — CHANGES_REQUESTED** [review]
Looks like the package-lock.json got uploaded on here, was this on purpose?  Other than that everything looks good.  Loads of changes here awesome job back-end! @WillisLi

**Review by fernando817mm — CHANGES_REQUESTED** [review]
Hey guys, it might be a good idea to drop the tables in reverse order of how they were created. Looking good though!

**Comment by WillisLi** [comment]
> Looks like the package-lock.json got uploaded on here, was this on purpose? Other than that everything looks good. Loads of changes here awesome job back-end! @WillisLi

Yep! That was on purpose, Ryan let us know that some stuff would be outdated, so we made the changes accordingly, and package-lock.json is part of that necessary change.

**Comment by WillisLi** [comment]
> Hey guys, it might be a good idea to drop the tables in reverse order of how they were created. Looking good though!

Sorry, which tables are you referring to? We checked the drop migrations and don't see any in particular that need to be fixed.

**Review by CptHappyHands — COMMENTED** [review]
Looks good to me!

**Review by JoeyMBrown — APPROVED** [review]
Awesome!  Thanks for the quick response!

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -25,10 +25,3 @@ jobs:
       - name: Install dependencies
         run:  npm install
       - run: npm run lint
-      - name: Run Tests
```
Comment (bradlylewis) [inline .github/workflows/ci.yml:28]: What are these 7 lines of code that were removed?

**Review by bradlylewis — APPROVED** [review]

**Comment by fernando817mm** [comment]
> > Hey guys, it might be a good idea to drop the tables in reverse order of how they were created. Looking good though!
> 
> Sorry, which tables are you referring to? We checked the drop migrations and don't see any in particular that need to be fixed.

exports.down in each migration file. I may be wrong but I thought that dropping the tables in reverse order from how they were created was best practice. Just asked Ash though and as long as you're consistent with the order of your tables it should be fine!

**Review by fernando817mm — APPROVED** [review]
Looks good!

File: .github/workflows/ci.yml (hunk 3 above)
Comment (WillisLi) [reply .github/workflows/ci.yml:28]: Just removed the SonarScan tests. Ryan approved these changes when reviewing what old setup code needed to be changed.

**Pull request merged**